### PR TITLE
Fix release app shortcut

### PR DESCRIPTION
**Pull request by takahirom**
## Overview (Required)
- I think it is mistake

### CODE REVIEW

File: app/src/main/res/xml-v25/shortcuts.xml [hunk 1]
```diff
@@ -7,7 +7,7 @@
         android:shortcutLongLabel="@string/favorite_title">
         <intent
             android:action="android.intent.action.VIEW"
-            android:targetPackage="io.github.droidkaigi.confsched2018.debug"
+            android:targetPackage="io.github.droidkaigi.confsched2018"
```
Comment (panpanini) [inline app/src/main/res/xml-v25/shortcuts.xml]: it looks like it should be corrected in the `release` flavour instead maybe 🤔  

https://github.com/DroidKaigi/conference-app-2018/blob/master/app/src/release/res/xml-v25/shortcuts.xml#L10

Comment (takahirom) [reply app/src/main/res/xml-v25/shortcuts.xml]: 🤔

Comment (takahirom) [reply app/src/main/res/xml-v25/shortcuts.xml]: Thanks we should change that 👍

**Comment by takahirom** [comment]
Fixed it!

**Comment by takahirom** [comment]
@panpanini 🙏

**Comment by takahirom** [comment]
@panpanini 🙏

**Pull request merged**